src/finit.c

```diff
@@ -368,12 +368,12 @@ int main(int argc, char *argv[])
 	};
 	uev_ctx_t loop;
 
-	/*
-	 * finit/init/telinit client tool uses /dev/initctl pipe
-	 * for compatibility but initctl client tool uses socket
-	 */
-	if (getpid() != 1)
-		return client(argc, argv);
+	if (getpid() != 1) {
+		fprintf(stderr, "Running Finit as anything other than PID 1 "
+			"is not supported.\n"
+			"The initctl tool is the new command line API.\n");
+		return 1;
+	}
```

Comment:
This is the only comment I have, everything else looks just great!  It is untested, but if we drop this code path entirely we should be able to run as a non-pid1 process monitor.  

However, since that particular use-case is untested, and wasn't part of the original scope for #136, I'll merge this as-is.